app/templates/components/public/social-links.hbs

```diff
@@ -19,11 +19,10 @@
   </div>
   {{#each this.socialLinks as |socialLink|}}
     {{#if (eq socialLink.normalizedName 'globe')}}
-      <div class="item">
-        <i class="fitted {{socialLink.normalizedName}} disabled icon"></i>
-        <div class="content">
-          <a href="{{socialLink.link}}" target="_blank" rel="noopener nofollow">{{socialLink.displayName}}</a>
-        </div>
+       <div class="m-1">
+        <a href="{{socialLink.link}}" class="ui small blue button" target="_blank" rel="noopener nofollow">
+          <div style="font-size: .8rem;">{{socialLink.displayName}}</div>
```

Comment:
also add css for long text or u can use textoverflow: ellipsis
like this
![image](https://user-images.githubusercontent.com/61561415/102684893-606c2500-4202-11eb-9394-7dc34a9b243f.png)